blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc

```diff
@@ -0,0 +1,28 @@
+
+= JBoss Fuse Tooling - Support of Global configurations

```

Comment:
FYI, you don't have to include the date in the filename anymore. Only the `page-date` metadata matters. This avoids renaming the file if it is published later than expected ;-)